### ISSUE TITLE
Can't reproduce results with KSC dataset

### ISSUE THREAD

**Issue by 0lykoek** [hantek/deeplearn_hsi#2]
I can't explain why, but when I used your KSC_SdA.py code (only changed the file roots), my final test error is above 20% and the Kappa index is close to 0.74, which is pretty far from the results in the "Deep Learning-Based Classification of Hyperspectral Data" article. Did you get those results (OA, AA, Kappa) with 3300 pretraining epochs, a pretraining learning rate of 0.6, 400000 training epochs, a training learning rate of 0.05, mini-batches of 100 and a single layer of 20 autoencoders or did you use other parameters? 

I got similar results as in the article with pavia_SdA.py.

Thanks.

**Comment by hantek** [comment]
That's exactly what I used. I suspect there might be sth wrong with the data. Did you tried with a data file downloaded fresh from the web?

**Comment by 0lykoek** [comment]
Yes, twice. The amelioration was about 1 point of percentage...

**Comment by hantek** [comment]
That's weird. If nothing is wrong in the data, could you check the files?

git clone the repo into another folder, and do `diff <repo_file_name> <your_file_name>`

If it turns out that nothing is changed, I'll try to rerun it myself.

**Comment by hantek** [comment]
Closing because no further response received.
